subsys/bluetooth/audio/ascs.c

```diff
@@ -1382,7 +1382,7 @@ static int ase_config(struct bt_ascs *ascs, struct bt_ascs_ase *ase,
 					BT_ASCS_RSP_CONF_REJECTED,
 					BT_ASCS_REASON_CODEC_DATA);
 
-			return err;
+			return 0;
```

Comment:
Why wouldn't we want to return the error here? 

If the client is using a write-with-response, it would not get an error response if something went wrong.